### PR TITLE
SetPowerMode board API section is titled SetPWM

### PR DESCRIPTION
**Pull request by randhid**
# Description

Please include a summary of the change and what is does. Please also include relevant motivation and context.

### CODE REVIEW

File: docs/components/board/_index.md [hunk 1]
```diff
@@ -420,7 +420,7 @@ err := myBoard.Status(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
-### SetPWM
+### SetPowerMode
```
Comment (npentrel) [inline docs/components/board/_index.md:423]: We go by the names defined in the proto: https://github.com/viamrobotics/api/blob/main/component/board/v1/board_grpc.pb.go#L33

are those changing?

Comment (randhid) [reply docs/components/board/_index.md:423]: The text under this is not decribing the correct proto method - it's describing https://github.com/viamrobotics/api/blob/80ccf2fe7a14d8bcdc3075b9e2b81f2458e6928f/component/board/v1/board_grpc.pb.go#L49

Comment (randhid) [reply docs/components/board/_index.md:423]: the actual `SetPWM` API is documented in line 880 of this file

Comment (npentrel) [reply docs/components/board/_index.md:423]: ahh - great catch - thanks!

**Review by npentrel — APPROVED** [review]

**Pull request merged**